dask/array/chunk.py

```diff
@@ -16,7 +16,7 @@
 try:
     from numpy import broadcast_to
 except ImportError:  # pragma: no cover
-    broadcast_to = npcompat.broadcast_to
+    from ..core import broadcast_to
```

Comment:
I'm not sure that this is what we want.  We're looking for the numpy `broadcast_to` function, not the dask array version.  However, I'm confused about our current behavior because I can't find `broadcast_to` in `numpy_compat`.  I wonder if @crusaderky can provide some information here.

Reply:
It was dropped from `npcompat` in PR ( https://github.com/dask/dask/pull/3429 ), which makes sense as `broadcast_to` was added in NumPy 1.10.0 and that PR made NumPy 1.11.0 our minimum requirement. So maybe the answer here is we should only keep the `try` part of this block and drop the rest.

cc @jcrist

Reply:
Yeah, should just drop the try all together, numpy 1.11.0 is our minimum requirement and `broadcast_to` should always be available there.

Reply:
Cool, so I've updated my pull request, just in case you feel like saving some of your keystrokes :).